docs/package.json

```diff
@@ -16,8 +16,8 @@
     "history": "^4.10.1",
     "lz-string": "^1.4.4",
     "marked": "^0.8.0",
-    "react": "^16.8.5",
-    "react-dom": "^16.12.0",
+    "react": "^16.13.0",
```

Comment:
Only update the docs since these include react as a dependency, not a peer dependency. That way we check whether everything still works in the latest version of React.